cluster/images/etcd/migrate/versions.go

```diff
@@ -126,6 +126,7 @@ func ParseEtcdVersionPair(s string) (*EtcdVersionPair, error) {
 
 // MustParseEtcdVersionPair parses a "<version>/<storage-version>" string to an EtcdVersionPair
 // or panics if the parse fails.
+// nolint:deadcode
```

Comment:
can you please add justification for all occurrences of `nolint:deadcode`? For example, 

```suggestion
// Part of public API, do not remove.
// nolint:deadcode
```

Reply:
Ack. That makes sense. I have added comments for all linter exemptions. 🙏🏽